Cargo.toml

```diff
@@ -15,6 +15,11 @@ build = "build.rs"
 cfg-if = "1.0"
 num-traits = { version = "0.2", default_features = false }
 uom = { version = "0.30", features = ["autoconvert", "f32", "si"] }
+serde = { version = "1.0.136", features = ["derive"] }
+
+[dependencies.schemars]
```

Comment:
Please keep this optional and have the feature in `starship` enable this (in starship: `config-schema = ["schemars", "starship-batter/schemars"]`).